Containers/apache/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN set -ex; \
         openssl \
         netcat \
         dpkg-dev \
+        curl \
```

Comment:
isn't curl already in the image?

Reply:
![grafik](https://user-images.githubusercontent.com/75573284/186661366-385bfbe9-6a09-4070-9908-71165d78f19b.png)

Reply:
all right!